lib/generators/draft/resource/resource_generator.rb

```diff
@@ -61,27 +61,27 @@ def generate_specs
 
   private
 
-    def golden_seven_routes
+    def golden_five_routes
       log :route, "RESTful routes"
 
       route <<-RUBY.gsub(/^      /, "")
 
         # Routes for the #{singular_table_name.humanize} resource:
 
         # CREATE
-        get("/#{plural_table_name}/new", { :controller => "#{plural_table_name}", :action => "new_form" })
-        #{skip_post? ? "get" : "post"}("/create_#{singular_table_name}", { :controller => "#{plural_table_name}", :action => "create_row" })
-
+        match("/post_#{singular_table_name}", { :controller => "#{plural_table_name}", :action => "create", :via => "#{skip_post? ? "get" : "post"}"})
+                
         # READ
-        get("/#{plural_table_name}", { :controller => "#{plural_table_name}", :action => "index" })
-        get("/#{plural_table_name}/:id_to_display", { :controller => "#{plural_table_name}", :action => "show" })
-
+        match("/#{plural_table_name}", { :controller => "#{plural_table_name}", :action => "index", :via => "get"})
+        
+        match("/#{plural_table_name}/:rt_#{singular_table_name}_id", { :controller => "#{plural_table_name}", :action => "show", :via => "get"})
+        
         # UPDATE
-        get("/#{plural_table_name}/:prefill_with_id/edit", { :controller => "#{plural_table_name}", :action => "edit_form" })
-        #{skip_post? ? "get" : "post"}("/update_#{singular_table_name}/:id_to_modify", { :controller => "#{plural_table_name}", :action => "update_row" })
-
+        
+        match("/patch_#{singular_table_name}/:rt_#{singular_table_name}_id", { :controller => "#{plural_table_name}", :action => "update", :via => "#{skip_post? ? "get" : "post"}"})
```

Comment:
How about `/modify_#{singular_table_name}/:rt_#{singular_table_name}_id` here?

Reply:
Sounds good to me.